core/amber/src/main/resources/application.conf

```diff
@@ -92,8 +92,7 @@ web-server {
 
 fault-tolerance {
     enable-determinant-logging = false
-    # [none, local, hdfs]
-    log-storage-type = "none"
+    log-storage-root-folder-uri = ""
```

Comment:
maybe "root-folder-uri" is not a natural name. especially folder is not a good term. how about just "log-storage-uri", which can be "file://", "hdfs://" or "s3://", etc.